lib/index.js

```diff
@@ -432,12 +436,14 @@ internals.type = function (value) {
 internals.at = function (error) {
 
     error = error || new Error();
-    const at = error.stack.replace(error.toString(), '').split('\n').slice(1).filter(internals.filterLocal)[0].match(/^\s*at \(?(.+)\:(\d+)\:(\d+)\)?$/);
-    return {
+    const stack = typeof error.stack === 'string' ? error.stack : '';
```

Comment:
Right. I'd rather ensure that we have a string though. Someone could create a new Error object and set the stack to something else. This will help keep `code` from breaking down if they do something stupid.